libs/ui/src/ui_strings/read_on_load.tsx

```diff
@@ -41,18 +41,18 @@ export function ReadOnLoad(props: ReadOnLoadProps): JSX.Element {
   const currentUrl = location?.pathname;
 
   const audioContext = useAudioContext();
-  const isAudioEnabled = audioContext?.isEnabled;
+  const isInAudioContext = !!audioContext;
```

Comment:
nit: Slight preference for `Boolean(...)` over `!!...` for boolean conversions

Reply:
+1 -- thanks, that does look more readable